### DISCUSSION TITLE
小白请教一下大佬，我是自己编译的CM520-79F的固件，为什么安装完成之后，在已安装的软件包里面有一些我并未选择的插件

### DISCUSSION THREAD

**Discussion by Fitz77777** [coolsnowwolf/lede#11089] [Q&A / 快速问答]
如题，例如
UnblockNeteaseMusic-Go  0.2.13-2
adbyby 0.2.13-2
我确定在LuCI ---> Applications下面没有选择这两个插件

**Comment by davidtall** [comment]
这个是依赖，Applications 取消选择后 依赖并不会自动取消。需要手动 编辑 .config 搜索把 UnblockNeteaseMusic-Go 和 adbyby  注释掉再编译
